### PR TITLE
added plugin for bintray

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [ ] Bugfix
- [X] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
added plugin for bintray, this plugin makes a bit easier

## :bulb: Motivation and Context
original bintray plugin requires some boilerplate

## :green_heart: How did you test it?
not yet, only with dryRun=false, so only locally so far.

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing

## :crystal_ball: Next steps
jar/aar signing

### CODE REVIEW

File: buildSrc/src/main/java/Config.kt [hunk 1]
```diff
@@ -40,12 +40,25 @@ object Config {
...
+        val version = "2.0.0-alpha01"
+        val description = "SDK for sentry.io"
+        val buildVersionCode = 20001
+        val website = "https://sentry.io"
+        val userOrg = "getsentry"
+        val repoName = "sentry-android"
+        val licence = "MIT"
+        val issueTracker = "https://github.com/getsentry/sentry-android/issues"
+        val repository = "https://github.com/getsentry/sentry-android"
+        val autoPublish = false // This boolean defines if the package will be published when uploaded. If this is false, the package will still be uploaded to bintray but you'll have to publish it manually.
```
Comment (marandaneto) [inline buildSrc/src/main/java/Config.kt]: @bruno-garcia what do you think? true or false? if false, we need to go on the website and press buttons

Comment (bruno-garcia) [reply buildSrc/src/main/java/Config.kt]: I'd just auto publish. It's important not to have the keys sourced to the shell automatically. (which means not using local.properties).
Let's discuss this offline

File: buildSrc/src/main/java/Config.kt [hunk 2]
```diff
@@ -40,12 +40,25 @@ object Config {
...
         val spotlessVersion = "3.25.0"
         val errorpronePlugin = "net.ltgt.gradle:gradle-errorprone-plugin:1.1.1"
     }
 
     object Sentry {
         val SENTRY_CLIENT_NAME = "sentry.java.android"
+        val group = "io.sentry"
+        val version = "2.0.0-alpha01"
+        val description = "SDK for sentry.io"
+        val buildVersionCode = 20001
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply buildSrc/src/main/java/Config.kt]: it needs to be an integer and sequential, it's common to use the pattern of the descriptive version, so `2.0.0-alpha01` = 20001 but open to change

File: sentry-android-core/build.gradle.kts [hunk 3]
```diff
@@ -89,3 +94,71 @@ dependencies {
...
+    artifactId = "sentry-android-core"
+}
+
+val VERSION_NAME = project.version.toString()
+val DESCRIPTION = Config.Sentry.description
+val SITE_URL = Config.Sentry.website
+val GIT_URL = Config.Sentry.repository
+val LICENSE = Config.Sentry.licence
+val DEVELOPER_ID = "marandaneto"
+val DEVELOPER_NAME = "Manoel Aranda Neto"
```
Comment (bruno-garcia) [inline sentry-android-core/build.gradle.kts]:
```suggestion
val DEVELOPER_NAME = "Sentry team and Contributers"
```

File: sentry-android-core/build.gradle.kts [hunk 4]
```diff
@@ -89,3 +94,71 @@ dependencies {
...
+    override = Config.Sentry.override
+    artifactId = "sentry-android-core"
+}
+
+val VERSION_NAME = project.version.toString()
+val DESCRIPTION = Config.Sentry.description
+val SITE_URL = Config.Sentry.website
+val GIT_URL = Config.Sentry.repository
+val LICENSE = Config.Sentry.licence
+val DEVELOPER_ID = "marandaneto"
```
Comment (bruno-garcia) [inline sentry-android-core/build.gradle.kts]:
```suggestion
val DEVELOPER_ID = "getsentry"
```

File: sentry-android-core/build.gradle.kts [hunk 5]
```diff
@@ -89,3 +94,71 @@ dependencies {
...
+val DEVELOPER_NAME = "Manoel Aranda Neto"
+val DEVELOPER_EMAIL = "maranda@sentry.io"
```
Comment (bruno-garcia) [inline sentry-android-core/build.gradle.kts]:
```suggestion
val DEVELOPER_EMAIL = "accounts@sentry.io"
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -16,5 +16,10 @@ To run the build and tests:
 ./gradlew build
 ```
 
+To publish it:
```
Comment (bruno-garcia) [inline CONTRIBUTING.md]: I'd suggest keeping this doc internal only (notion).
No one outside of Sentry will be publishing these.

File: buildSrc/src/main/java/Config.kt [hunk 7]
```diff
@@ -40,21 +40,47 @@ object Config {
...
         val jacocoVersion = "0.8.5"
+        val spotless = "com.diffplug.gradle.spotless"
         val spotlessVersion = "3.25.0"
         val errorpronePlugin = "net.ltgt.gradle:gradle-errorprone-plugin:1.1.1"
     }
 
     object Sentry {
         val SENTRY_CLIENT_NAME = "sentry.java.android"
+        val group = "io.sentry"
+//        TODO: change version to publish new version
```
Comment (bruno-garcia) [inline buildSrc/src/main/java/Config.kt]: This is something `craft` can do for us.

**Pull request merged**